petab/core.py

```diff
@@ -46,7 +46,11 @@ def get_visualization_df(visualization_file: str) -> pd.DataFrame:
     Returns:
         Visualization DataFrame
     """
-    return pd.read_csv(visualization_file, sep="\t", index_col=None)
+    try:
+        vis_spec = pd.read_csv(visualization_file, sep="\t", index_col=None)
+    except pd.errors.EmptyDataError:
+        vis_spec = pd.DataFrame()
+    return vis_spec
```

Comment:
maybe include warning? Usually users probably do not intend to have empty visu files